python/requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy==1.20.3
+numpy==1.21.0
```

Comment:
Is it good to hard-pin these, or could we opt for the compatible releases as `numpy~=1.21.0`?

Reply:
All good, happy to approve.